oneview-sdk-java-lib/src/main/java/com/hp/ov/sdk/util/SdkUtils.java

```diff
@@ -117,11 +116,7 @@ private void validateApiVersion(final int actualVersion, final int expectedVersi
             logger.info("API version : " + "You are trying to connect to appliance verion: " + actualVersion
```

Comment:
We could change the versions equality test to a less than or equal. Doing that we would still have a log entry message when the actual version is less than the expected version (which is a valid usage of the SDK).